.github/workflows/develop-ci.yml

```
@@ -0,0 +1,133 @@
+
+      - name: 테스트 실패 시 코드에 표시
+        uses: mikepenz/action-junit-report@v4
+        if: always()
+        with:
+          report_paths: '**/build/test-results/test/TEST-*.xml'
+          check_name: 테스트 상세 결과
+
+  notify-discord:
+    needs: test
+    runs-on: ubuntu-latest
+    if: ${{ always() && secrets.DISCORD_WEBHOOK != '' }}
```

actionlint (1.7.10)

context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)